src/Npgsql.NodaTime/NpgsqlNodaTimeExtensions.cs

```diff
@@ -15,7 +15,7 @@ public static class NpgsqlNodaTimeExtensions
     /// <param name="mapper">The type mapper to set up (global or connection-specific)</param>
     public static INpgsqlTypeMapper UseNodaTime(this INpgsqlTypeMapper mapper)
```

Comment:
As discussed offline, maybe we should just call GlobalTypeMapper.AddTypeInfoResolver here too? That would solve the isNpgsqlDbTypeInferredFromClrType issue (once I've fixed us not picking those up in the global type mapping options)

Reply:
As discussed offline, we're not changing behavior/breaking anything compared to 7.0: plugin registrations on data sources already didn't affect inference (which goes through GlobalTypeMapper).

My personal preference is not doing something preemptively here - I'd wait and see if people actually complain. I think the ultimate place we want to get to one day is another inference API on DbDataSource rather than on NpgsqlParameter.